configuration/metrics.nix

```diff
@@ -62,6 +63,10 @@
     }
   '';
 
+
+  sops.secrets.grafanaClientSecret = {};
+  sops.secrets.grafanaClientSecret.owner = config.users.users.grafana.name;
```

Comment:
Use something similar to [outline.nix](https://github.com/83-Atelier/clematis/blob/master/configuration/outline.nix#L14-L17) here.